TCMSApp/src/client/app/defects/defects.controller.spec.js

```diff
@@ -1,4 +1,42 @@
 /* jshint -W117, -W030 */
-describe('DefectsController', function() {
+describe('tests Defects', function() {
+    //this.timeout(15000);
+    var controller;
+    beforeEach(function () {
+            bard.appModule('app.defects');
+            bard.inject('$controller', '$rootScope', '$httpBackend', '$state', 'apiUrl');
+        });
 
+    beforeEach(function () {
+            var scope = $rootScope.$new();
+            var state = {};
+            state.$current = {};
+            state.$current.name = 'defects';
+            controller = $controller('DefectsController', {$scope: scope, $state: state});
+            var httpBackend = $httpBackend;
+            httpBackend.when('GET', apiUrl.defects).respond(
+                [
+                    {
+                        '_id':'56b0827025c2c3cc1146fc05',
+                        'name':'ErrorName14587',
+                        'assignedTo':'',
+                        'dateOfDefectCreation':'2016-02-02T10:17:34.410Z',
+                        'priority':'Normal',
+                        'description':'rfrfrfnhvnslhrrhzhvsc\nhvwhnghnecw45645cshgkvmm',
+                        'stepsToReproduce':['gfhdjhnbjnjnyvnynhnjnkynk'],
+                        '__v':0
+                    }
+                ]
+            );
+            httpBackend.flush();
+            $rootScope.$digest();
+        });
+
+    bard.verifyNoOutstandingHttpRequests();
+
+    describe('Defects Controller', function () {
+        it('should be created successfully', function () {
+            expect(controller).to.be.defined;
+        });
+    });
```

Comment:
If yo wish to move somewhere from defects, it can be the separate task (not for you)